README.md

```diff
@@ -1,14 +1,21 @@
 ![The App Business](https://github.com/theappbusiness/MasterFastfile/blob/master/MasterFastfile.png)
 
 # MasterFastfile
- To setup your project to use the TAB master fastfile navigate to your project route and run the following command
+ To setup your project to use the TAB master fastfile navigate to your project root and run the following command
 
 ```
 curl https://raw.githubusercontent.com/theappbusiness/MasterFastfile/master/setup.sh | sh
 ```
 
 Then setup your environments using the `.env` files created for you. To complete the setup just add the missing variable values.
 
+To use the Master Fastfile add the following command to your Fastfile
+
+```
+import_from_git(url: 'https://github.com/theappbusiness/MasterFastfile.git', branch: 'tags/1.5.0', path: 'Fastfile')
+```
+For more detailed instructions see our wiki [here](https://github.com/theappbusiness/MasterFastfile/wiki)
```

Comment:
I'd have this as:

`For more detailed instructions, [see our wiki] (https://github.com/theappbusiness/MasterFastfile/wiki)`